src/sdk.ts

```diff
@@ -573,8 +574,9 @@ s	 * @example
 				return response || data.success;
 			}
 
-			throw new Error(
-				`Request ${url} returned an error! response=${JSON.stringify(data)}`
+			throw new PaddleRequestError(
```

Comment:
Please add a couple of new expectations in the unit tests that test the error cases.

Reply:
Done in https://github.com/avaly/paddle-sdk/pull/85/commits/9cc168b27c3cf69aef984571e72a8e28207937ec